tests/fixtures.py

```diff
@@ -231,6 +222,30 @@ def plan_validator(module_path, inventory_paths, basedir, tf_var_files=None,
   return summary
 
 
+def validate_plan_object(expected_value, plan_value, relative_path, relative_address):
+  # dictionaries / objects
+  if isinstance(expected_value, dict):
```

Comment:
I'd check here if both sides are dicts (expected_value and plan_value), just to cover the case where they're not and you fallback to the default case where you compare with `==` (which will probably fail with a nice exception)

Idem for lists below